### ISSUE TITLE
[FEATURE REQUEST] - Remove the requirement to put the product key on the AME Wizard

### ISSUE THREAD

**Issue by instasio** [Atlas-OS/Atlas#986]
### What is your feature request regarding to?

AtlasOS

### Is your feature request related to a problem? Please describe.

Remove the requirement to put the product key on the AME Wizard

### Describe the solution you would like.

Remove the requirement to put the product key on the AME Wizard

### Describe alternatives you have considered.

Remove the requirement to put the product key on the AME Wizard

### Requisites

- [X] This is not a support issue or a question. For any support, questions or help, join our [Discord server](https://discord.atlasos.net).
- [X] I performed a [cursory search of the issue tracker](https://github.com/Atlas-OS/Atlas/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I understand that not filling out this template will lead to the issue being closed.

### Additional context.

Remove the requirement to put the product key on the AME Wizard

**Comment by JayXTQ** [comment]
> Atlas prevents you from activating after applying the playbook, so we might as well force people to do it beforehand.

Not true, there is ways to get around this, which I have made guides on before... but it can get scarily close to telling people how to pirate so I guess you could be correct. Haven't tried opening the activation page on Atlas yet tbh
